src/OptionT.ts

```diff
@@ -353,6 +405,17 @@ class Some<T> extends OptionT<T> {
   }
 
   unwrap(message?: string): T {
+    if (!this.hasBeenInspected) {
+      throw new Error(
+        'Called unwrap without first checking if it was safe to do so. Please verify' +
+          ' that the OptionT in question is a `Some` value before calling this function.'
+      );
+    }
+    return this.value;
+  }
+
+  forceUnwrap(message?: string): T {
+    console.warn('Warning: called forceUnwrap on a `Some` value.  This is not recommended usage.');
```

Comment:
We might should add error/warning codes to our messages - something ive seen other projects do is actually make their codes urls to documentation, so its easy to google but also to get more information about the problem and possible solutions.

I think steering them to use `unwrap` or `unwrapOr` might be good here too.

Reply:
>We might should add error/warning codes to our messages[...]

This is an interesting idea.  I'd like to see an example of this if you have one.  Is this similar to what Rust does with their compiler error codes?  If you're not familiar they basically have a giant error page with further explanations for all of the error codes in their docs somewhere.  I never get there other than by googling an error code, so I'm not exactly sure where they are.

Only concern here is that those URL links are something extra we'd need to maintain if the docs location ever changed.  If this is going to end up being a slightly more involved changed then maybe we should make an issue about it and approach it in a separate PR?  Not sure there.

>I think steering them to use `unwrap` or `unwrapOr` might be good here too.

Agree, will make this change as well.

Reply:
I think for now lets just create a specific error code that we can include in documentation - something like `nullshield:force_unwrap_warning: called forceUnwrap on ...` so that its easy to copy and paste into google.